### ISSUE TITLE
Have you written any test cases for ask-sdk using mocha or chai ?

### ISSUE THREAD

**Issue by paigurunath** [rahul71/alexa-unit-test-simple-mock#1]
Have you written any test cases for ask-sdk using mocha or chai ?

**Comment by rahul71** [comment]
yes, got a chance to work on it today. Do you need it. I can update on my github this weekend

**Comment by blutag-dev** [comment]
@paigurunath I updated my simple unit test, which mocks using nock and uses mocha and chai here
https://github.com/blutag-dev/alexa-unit-test-simple-mock
